horace_core/admin/cache.py

```diff
@@ -82,13 +81,11 @@ def read_chunk(filename,start,size,buf_size,progress,n_workers):
     if progress:
         prog_rep = progress_rep(size,n_workers)
         
-    fh = open(filename,'rb')
-    if fh<0:
-        raise ValueError("Can not open file: "+filename)
+    fh = open(filename,'rb',buffering=0)
     fh.seek(start,0)
```

Comment:
Unnecessary

Reply:
? opening file? How?

Reply:
Then the seek should be in the with block

Reply:
I just realised the original comment got cut off it should read
> Unnecessary `whence` argument, 0 is the default and regardless this should be using the [`enum`s defined in `os`](https://docs.python.org/3/library/io.html#io.IOBase.seek)